internal/webhook/v1alpha1/paasconfig_webhook.go

```diff
@@ -232,6 +233,31 @@ func validateConfigCapabilities(capabilities v1alpha1.ConfigCapabilities, rootPa
 	return allErrs
 }
 
+func validateConfigCapabilityNames(spec v1alpha1.PaasConfigSpec, rootPath *field.Path) field.ErrorList {
+	if spec.Validations == nil {
+		return nil
+	}
+	childPath := rootPath.Child("capabilities")
+	validationRE := spec.Validations.GetValidationRE("paasConfig", "capabilityName")
+	if validationRE == nil {
+		return nil
+	}
+
+	var allErrs field.ErrorList
+
+	for name := range spec.Capabilities {
+		if !validationRE.Match([]byte(name)) {
+			allErrs = append(allErrs, field.Invalid(
+				childPath.Key("name"),
```

Comment:
```suggestion
				childPath.Key(name),
```

shouldnt' this be a reference to the name var?